code/game/jobs/job/security.dm

```diff
@@ -249,6 +249,8 @@
 	departments = SIMPLEDEPT(DEPARTMENT_SECURITY)
 	department_flag = ENGSEC
 	faction = "Station"
+	alt_titles = list("Investigator Intern", "Warden Cadet")
```

Comment:
investigator intern needs an age minimum set to 24/25 years old

Reply:
The medical intern does not have this, despite similar age restrictions existing for its jobs. Do you want this for that as well?